src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -475,4 +477,50 @@ protected override void Dispose(bool disposing)
 
         base.Dispose(disposing);
     }
+
+    private void ApplySpecificationConfigurationKeys(IConfiguration configuration)
+    {
+        if (configuration.TryGetBoolValue(EmitOverFlowAttributeConfigKey, out this.EmitOverflowAttribute))
+        {
+            OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent("Overflow attribute feature enabled via configuration.");
+        }
+
+        if (configuration.TryGetBoolValue(ReclaimUnusedMetricPointsConfigKey, out this.ReclaimUnusedMetricPoints))
+        {
+            OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent("Reclaim unused metric point feature enabled via configuration.");
+        }
+
+        if (configuration.TryGetStringValue(ExemplarFilterConfigKey, out var configValue))
```

Comment:
A question:

This is NOT inside an `#if EXPOSE_EXPERIMENTAL_FEATURES` block. That means users will be able to enable exemplars in stable builds using the envvar key. Is this something we want?

Reply:
I think it should be inside #if expose experimental block, and use the official name as well.
https://github.com/open-telemetry/opentelemetry-dotnet/pull/5412/files#r1513355039

Reply:
Updated